docs/InputsFields.md

```diff
@@ -1,3 +1,8 @@
+---
+layout: default
+title: "admin-on-rest - Input and Field Components"
```

Comment:
We should mutualize the `admin-on-rest - ` title part directly in the main layout.